docs/source/tutorials/object_detection.ipynb

```diff
@@ -0,0 +1,357 @@
+    "\n",
+    "<div class=\"alert alert-info\">\n",
+    "Quickstart\n",
+    "<br/>\n",
+    "    \n",
+    "Already have `labels` and `predictions` in the proper format? Just run the code below to find label issues in your object detection dataset.\n",
+    "\n",
+    "\n",
+    "<div  class=markdown markdown=\"1\" style=\"background:white;margin:16px\">  \n",
+    "    \n",
+    "```python\n",
+    "\n",
+    "from cleanlab.object_detection.rank import find_label_issues\n",
+    "from cleanlab.object_detection.filter import get_label_quality_scores\n",
+    "\n",
+    "# To get boolean vector of label issues for all images\n",
+    "is_label_issue = find_label_issues(labels, predictions)\n",
+    "\n",
+    "# To get label quality scores for all images\n",
+    "label_quality_scores = get_label_quality_scores(labels, predictions)\n",
+    "    \n",
+    "    \n",
+    "```\n",
+    "\n",
+    "</div>\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8d552ab9",
+   "metadata": {},
+   "source": [
+    "# 1. Install required dependencies and download data\n",
+    "You can use `pip` to install all packages required for this tutorial as follows\n",
+    "```\n",
+    "!pip install cleanlab\n",
+    "!pip insrall matplotlib\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c90449c8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!wget -nc 'https://cleanlab-public.s3.amazonaws.com/ObjectDetectionBenchmarking/tutorial_obj/predictions.pkl'\n",
+    "!wget -nc 'https://cleanlab-public.s3.amazonaws.com/ObjectDetectionBenchmarking/tutorial_obj/labels.pkl'\n",
+    "!wget -nc 'https://cleanlab-public.s3.amazonaws.com/ObjectDetectionBenchmarking/tutorial_obj/example_images.zip' && unzip -q -o example_images.zip"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "df8be4c6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pickle\n",
+    "\n",
+    "from cleanlab.object_detection.rank import get_label_quality_scores, issues_from_scores, visualize\n",
+    "from cleanlab.object_detection.filter import find_label_issues"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2506badc",
+   "metadata": {},
+   "source": [
+    "# 2. Format data, labels and model predictions\n",
+    "\n",
+    "In object detection datasets, each given `label` is a made up of bounding box coordinates and a class label. A model `prediction` is also made up of a bounding box and class label as well as the model confidence in the prediction (i.e. probability the prediction is correct). For both `get_label_quality_scores` and `find_label_issues` cleanlab requires a list of given labels and a list of model predictions for all examples.\n",
+    "\n",
+    "The expected format of these `labels` and `predictions` is modeled after what common models like mmdet (link) and detectron2 (link) expect. \n",
```

Comment:
Clarify:

Here are the 5 possible class labels in our dataset: …

Also clarify we are representing each class with an integer 0,1,...,K-1